modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,27 @@
 [id="installation-osp-kuryr-known-limitations_{context}"]
 = Known limitations of installing with Kuryr
 
-There are known limitations when using Kuryr SDN:
+There are known limitations when using Kuryr SDN, depending on the
+{rh-openstack} version:
 
 * An Amphora load balancer VM is deployed per OpenShift Service with
-the default Octavia load balancer driver (Amphora driver). If the environment is
-resource constrained, creating a large amount of Services could be a problem.
-* Depending on the Octavia version, UDP listeners are not supported. This means
-that OpenShift UDP Services are not supported.
-* There is a known limitation of Octavia not supporting listeners on different
-protocols, like UDP and TCP, on the same port. Thus, Services exposing the same
-port for different protocols are not supported.
+the default Octavia load balancer driver (Amphora driver). In a
+resource-constrained environment, creating too many Services can cause
+problems.
+* Octavia {rh-openstack} version 13 does not support UDP listeners. This means
+that OpenShift UDP services are not supported. UDP listeners a re supported
+from Octavia {rh-openstack} version 16 and later.
+* Octavia {rh-openstack} version 13 cannot listen to multiple protocols,
+like UDP and TCP, on the same port. Service that expose the same port to
+different services are not supported. Listeners that use different protocols
+are supported in {rh-openstack} version 16 and later. Note this only applies
+to Octavia amphora driver. OVN Octavia driver still does not support listeners
+that use different protocols in {rh-openstack} version 16.
 * Due to the above UDP limitations of Octavia, Kuryr forces Pods to use TCP
-for DNS resolution. This is set with the `use-vc` option in `resolv.conf`. This
-might be a problem for Pods running Go applications compiled with the `CGO_DEBUG`
-flag disabled, as that uses the `go` resolver that only leverages UDP and is not
-considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a
-problem also for musl-based containers as its resolver does not support the
-`use-vc` option. This includes images built from `alpine`.
+for DNS resolution in {rh-openstack} version 15 or earlier, or if OVN Octavia
```

Comment:
Would it make sense to rephrase or rearrange this whole section so that it's in terms of <= version 15 vs. version 16? I could give that a shot, if so.

Reply:
First point is independent of the OSP version. 2nd and 3rd point are dependent on that. And the last one is a consequence of those. Perhaps that could be added in a follow up PR. Feel free to open that and ping me to take a look. What do you think?